xml/docu_styleguide.language.xml

```diff
@@ -204,9 +213,10 @@
    </listitem>
    <listitem>
     <para>
-     Use a trailing slash to distinguish between a file and a
+     Use a trailing slash (for example, <filename>/etc/sysconfig/
+     </filename>) to distinguish between a file and a
      directory. If the distinction is important in the current context, in
-     addition comment on it in the accompanying or surrounding running text.
+     addition comment on it in the running text.
```

Comment:
In SUMA, after some discussion, we have started using a form like this:

"In the `/etc/hosts/` directory ..." 

That way you get the trailing slash (which makes sense to most Linux users), but still use the word "directory" for all those who aren't so used to looking carefully at these things.

Reply:
I created a separate PR for this - see https://github.com/SUSE/doc-styleguide/pull/164